store/index.js

```diff
@@ -1,7 +1,8 @@
-import axios from 'axios'
+import { initMiddleware } from './utils'
 
 export const state = () => ({
   nodeStatus: {},
+  middleware: null,
```

Comment:
```suggestion
  middleware: initMiddleware(),
```
I don't think that plugins are needed to run a sync code